questionpy_sdk/resources/__init__.py

```diff
@@ -0,0 +1,5 @@
+from importlib.resources import as_file, files
+
+
+with as_file(files(__package__)) as directory:
+    EXAMPLE_PACKAGE = directory / "example.zip"
```

Comment:
Mit Blick auf die Doku von [`as_file`](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.as_file) macht mich das stutzig:
> Exiting the context manager cleans up any temporary file created when the resource was extracted from e.g. a zip file.

D.h. sollte questionpy-sdk selbst in einer Zip-Datei liegen, sollte das nicht funktionieren. Stattdessen müsste der Context Manager jedes mal aufgemacht werden, wenn das Beispielpaket gebraucht wird.

Mein `Path(inspect.getfile(pkg)).parent` in `commands.package` hat übrigens die gleiche Beschränkung. Du kannst ja mal schauen, ob du das ohne viel Aufwand auch so umgesetzt kriegst, aber da geht es halt um ein ganzes Paket bzw. Verzeichnis.

Reply:
Das stimmt. Ich hab gerade mit Jan gesprochen. Ich nehme diesen PR an und er kümmert sich in einem neuen PR um beide Probleme.